### PR TITLE
LeaderBoard - Feature branch

### PR DESCRIPTION
**Pull request by Emmykage**
Created Leader Board Skeletal structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
-# LeaderBoard-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> API TBD
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly add a description of the project to make your readme more professional.

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+## Built With
+
+- HTML, CSS and Javascript
+- Git and Github
+
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly include web-pack as this project is built upon it also.

File: README.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+## Live Demo (if available)
+
+https://emmykage.github.io/
+
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly update or remove the live demo link as this links to your portfolio and not to the current project.

File: README.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+  None
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly update or remove this header as you've specified there is no prerequisite needed to run the project.
- Also, I think the user could need a visual studio code or any other code editor they may prefer.

File: README.md [hunk 5]
```diff
@@ -1 +1,66 @@
...
+👤 Menanya Morris
+
+- GitHub: [@emmykage](https://github.com/emmykage)
+- Twitter: [@emmykage](https://twitter.com/emmykage)
+- LinkedIn: [Menanya Morris](https://www.linkedin.com/in/morris-menanya-a51985104/)
+
+
+## 🤝 Contributing
+https://github.com/Emmykage/Capstone-Project/issues
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly update the issue page to this project, as the one here now is a different one.

File: README.md [hunk 6]
```diff
@@ -1 +1,66 @@
...
+Thanks to everyone who contributed
+
```
Comment (zieeco) [inline README.md:52]: **[REQUIRED CHANGES]**

- Please, kindly watch out for typos as this doesn't make you appear professional.

File: README.md [hunk 7]
```diff
@@ -1 +1,66 @@
...
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- i want to Ackowledge Cindy Shin whose original design was used to make this website
+
+
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly update the acknowledgment as this is referring to the Cindy design used in your html-css capstone project which is not relevant to this project.

File: dist/index.html [hunk 8]
```diff
@@ -1,9 +1,43 @@
...
+            <ul>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+                <li> Name: 200</li>
+
+
```
Comment (zieeco) [inline dist/index.html]: **[REQUIRED CHANGES]**

- Please, the list items should be added dynamically using javascript. Please populate this part using javascript.

File: src/style.css [hunk 9]
```diff
@@ -1,8 +1,63 @@
...
+  height: 40px;
+  margin-bottom: 8px;
+}
+
+.btn {
+  padding: 8px;
+  display: flex;
+  align-items: center;
+  height: 28px;
+}
```
Comment (zieeco) [inline src/style.css]: **[REQUIRED CHANGES]**

- Please, kindly adjust your styling to match the wireframe below:

*wireframe*
matchhh

*your design*
your_wireframe

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @Emmykage, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter error :heavy_check_mark: 
- Used gitflow correctly :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Emmykage     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**